### PR TITLE
N5222B: Support multiple attenuators_options

### PR DESCRIPTION
**Pull request by cjchua**
The previous version only recognized option 419 as an option with attenuation. This proposed change checks for all other possible options that also add source/receiver attenuation.

### CODE REVIEW

File: qcodes/instrument_drivers/Keysight/N5222B.py [hunk 1]
```diff
@@ -11,6 +11,7 @@ def __init__(self, name: str, address: str, **kwargs: Any):
                          nports=4,
                          **kwargs)
 
-        options = self.get_options()
-        if "419" in options:
-            self._set_power_limits(min_power=-90, max_power=13)
+        attenuators_options = {'217', '219', '220', '417', '419', '420'}
+        options = set(self.get_options())
+        if attenuators_options.intersection(options):
+            self._set_power_limits(min_power=-95, max_power=13)
```
Comment (jenshnielsen) [inline qcodes/instrument_drivers/Keysight/N5222B.py:17]: Is is correct that all these options results in the same power limits ?

Comment (cjchua) [reply qcodes/instrument_drivers/Keysight/N5222B.py:17]: Hi Jens,

Yes, I think so. Please see the following links for options on the various PNA models:

https://na.support.keysight.com/vna/help/latest/Support/Configurations.htm#N522x
https://na.support.keysight.com/pna/help/latest/Support/Configurations.htm

It states that the attenuation is the same for these options, but varies for every PNA model.

**Comment by jenshnielsen** [comment]
bors merge

**Review by jenshnielsen — APPROVED** [review]

**Comment by jenshnielsen** [comment]
bors merge

**Pull request merged**